src/app/services/standup.ts

```diff
@@ -0,0 +1,21 @@
+import { standupUpdateType, userDetails } from '@/interfaces/standup.type';
+import { api } from './api';
+
+export const standupApi = api.injectEndpoints({
+    endpoints: (build) => ({
+        addStandup: build.mutation<void, standupUpdateType>({
+            query: (standupUpdate) => ({
+                url: '/progresses',
+                method: 'POST',
+                body: standupUpdate,
+            }),
```

Comment:
what about the headers, how will you send the auth tokens?

Reply:
in pages/standup/index.js file, we are checking there itself if a user is authenticated or not then only he will be able to fill out the standup form. why do we need to send auth tokens in headers and also I am able to send request successfully to the backend.

Reply:
i am not sure about this approach so please take other's opinion